### ISSUE TITLE
Add PhoneGap version (device.phonegap) to sample

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#60]
```
see http://docs.phonegap.com/en/1.3.0/phonegap_device_device.md.html#Device


```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 15 Jan 2012 at 8:04

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 28 Jan 2012 at 7:00
- Added labels: **Milestone-Release1.2.7**

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 2 Feb 2012 at 5:47
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Released with 1.2.7 February 7, 2012
```

Original comment by `paul.beu...@gmail.com` on 7 Feb 2012 at 4:10
- Changed state: **Fixed**
